joomla/joomla-cms
# Remove forgotten 'card-link' parameter from field and blog sample data

Pull Request for Issue #44088 .

### Summary of Changes
Removed an unused parameter.


### Testing Instructions
Create a module and check if the parameter "Category Link" is displayed when you display the category

### Actual result BEFORE applying this Pull Request
No field Category Link is showed

### Expected result AFTER applying this Pull Request
Field Category Link is present
![grafik](https://github.com/user-attachments/assets/0a5d60ed-363f-442c-8eb5-ac261a30c66c)

### Link to documentations
Please select:
